tests/check_framework/test_templates.py

```diff
@@ -0,0 +1,39 @@
+from django.core.checks.templates import E001
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
```

Comment:
I'd make the settings more minimal, omitting things like 'context_processors' which aren't important for the test.